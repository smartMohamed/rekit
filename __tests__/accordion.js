import React from 'react'
import Enzyme, {shallow} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Accordion from '@/components/accordion'

Enzyme.configure({ adapter: new Adapter() })

describe('Accordion component', () => {
  it('Should render component and match snapshot', () => {

  const wrapper = shallow(
    <Accordion placeholder='Title' open={false} />
  )

  expect(wrapper.html()).toMatchSnapshot()
  })

  it('Should render component and match snapshot', () => {
  
    const wrapper = shallow(
      <Accordion placeholder='Title' open={true} >Some content</Accordion>
    )
    expect(wrapper.html()).toMatchSnapshot()
    expect(wrapper.find('.fk-accordion__icon').hasClass('fk-accordion__icon--open'))
  })
})