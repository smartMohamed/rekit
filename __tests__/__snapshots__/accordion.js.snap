// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion component Should render component and match snapshot 1`] = `"<section class=\\"fk-accordion\\"><div class=\\"fk-accordion__header fk-flex fk-flex--justify-space-between fk-flex--align-center\\" justify=\\"space-between\\" align=\\"center\\"><h5 class=\\"fk-accordion__title fk-title fk-title--xs\\" color=\\"\\" type=\\"\\" style=\\"font-size:xs\\"> Title </h5><span><svg viewBox=\\"0 0 38 38\\" class=\\"fk-icon fk-accordion__icon\\"><path fill-rule=\\"evenodd\\" d=\\"M35.614 16.623h-14.24V2.369a2.373 2.373 0 1 0-4.746 0v14.254H2.388a2.376 2.376 0 0 0 0 4.751h14.24v14.255a2.373 2.373 0 1 0 4.746 0V21.374h14.24a2.376 2.376 0 0 0 0-4.751z\\"></path></svg></span></div></section>"`;

exports[`Accordion component Should render component and match snapshot 2`] = `"<section class=\\"fk-accordion\\"><div class=\\"fk-accordion__header fk-flex fk-flex--justify-space-between fk-flex--align-center\\" justify=\\"space-between\\" align=\\"center\\"><h5 class=\\"fk-accordion__title fk-title fk-title--xs\\" color=\\"\\" type=\\"\\" style=\\"font-size:xs\\"> Title </h5><span><svg viewBox=\\"0 0 38 38\\" class=\\"fk-icon fk-accordion__icon fk-accordion__icon--open\\"><path fill-rule=\\"evenodd\\" d=\\"M35.614 16.623h-14.24V2.369a2.373 2.373 0 1 0-4.746 0v14.254H2.388a2.376 2.376 0 0 0 0 4.751h14.24v14.255a2.373 2.373 0 1 0 4.746 0V21.374h14.24a2.376 2.376 0 0 0 0-4.751z\\"></path></svg></span></div><div class=\\"fk-accordion__content-frame\\"><div class=\\"fk-accordion__content\\">Some content</div></div></section>"`;
